sunpy/net/dataretriever/sources/kanzelhohe.py

```diff
@@ -65,10 +65,15 @@ def _get_url_for_timerange(self, timerange, **kwargs):
         #Checking if value is close enough to a wavelength value.
         #Converting from one unit to other introduces precision errors.
         try:
-            da = []
+            #The isclose function in numpy can only check two arrays if they
+            #are close to each other within some precision. Standalone values such
+            #as int, doubles etc can't be checked that way. In order to do that, we put the
+            #two indiviual values in two seperate arrays da and db and then apply
+            #isclose on both those arrays.
+            da = list()
```

Comment:
The names don't mean a thing. Just two arrays A and B. I couldn't think of any better names. Just like you use variables x & y to explain something to someone, I picked da and db. If you want me to change them I will. Please do suggest some names as well, I can't think of anything meaningful.